packages/algob/src/lib/account.ts

```diff
@@ -71,3 +71,23 @@ export function mkAccountIndex (accountList: Account[]): Accounts {
   }
   return out;
 }
+
+export async function loadKMDAddresses (host: string, token: string, port: number,
+  walletName: string, password: string): Promise<AccountSDK[]> {
+  const kmdclient: Kmd = await new Kmd(token, host, port);
+  let walletid = '';
+  const wallets = (await kmdclient.listWallets()).wallets;
+  for (const arrayItem of wallets) {
+    if (arrayItem.name === walletName) {
+      walletid = arrayItem.id;
+    }
+  }
+  const wallethandle = (await kmdclient.initWalletHandle(walletid, password)).wallet_handle_token;
+  const address = await kmdclient.listKeys(wallethandle);
+  const mkaccount: AccountSDK[] = [];
```

Comment:
why this variable is called `mkaccount`?

Reply:
ok, i will rename it to accounts.